lib/modules/manager/vendir/__fixtures__/empty-directories.yml

```diff
@@ -0,0 +1,3 @@
+apiVersion: vendir.k14s.io/v1alpha1
```

Comment:
Please create a `readme.md` file in this location: `lib/modules/manager/vendir/readme.md` and explain the limitations of the new Vendir manager, like this:

> The Vendir manager can only updates dependencies of the `helmChart` type.